### ISSUE TITLE
Trouble getting singularity image to work

### ISSUE THREAD

**Issue by sjw42** [HBClab/NiBetaSeries#282]
**Describe the bug**
I'm sure this is an issue with our hpc system and not a bug, but I thought that someone may have insight into what is going wrong. I sincerely apologize if submitting here as an issue is not appropriate. I built a singularity container for NiBetaSeries v0.4.2 and got the error pasted below when trying to use it:
Traceback (most recent call last):
  File "/opt/miniconda-latest/bin/nibs", line 8, in <module>
    sys.exit(main())
  File "/opt/miniconda-latest/lib/python3.7/site-packages/nibetaseries/cli/run.py", line 237, in main
    atlas_img=os.path.abspath(opts.atlas_img),
  File "/opt/miniconda-latest/lib/python3.7/posixpath.py", line 378, in abspath
    path = os.fspath(path)
TypeError: expected str, bytes or os.PathLike object, not NoneType

**To Reproduce**
I used the command pasted below to point to the container and run the program. I thought that passing the cleanenv flag would avoid conflicts with any environment variables. I also tried unset PYTHONPATH with no luck. 
 
singularity run --cleanenv /gpfs/group/sjw42/default/sjw42_collab/sw/singularity/nibetaseries/nibetaseries-v0.4.2.simg \
nibs -c white_matter \
--participant-label 01 \
-w /gpfs/group/sjw42/default/ASH/SEMA/work \
/gpfs/group/sjw42/default/ASH/SEMA/bids \
fmriprep \
/gpfs/group/sjw42/default/ASH/SEMA/bids/derivatives \
participant

**Expected behavior**

**Desktop (please complete the following information):**
Redhat

**nibetaseries version**
v0.4.2

**Additional context**
Thanks for any thoughts or suggestions for troubleshooting, and sorry again if I should be seeking help elsewhere.

**Comment by jdkent** [comment]
Hi @sjw42,

Thanks for opening an issue, I see the problem arises when one does not specify `--atlas-img` and/or `--atlas-lut` in the command. Your use case is one that should work, so I will change that code and release a new version shortly (e.g., today or tomorrow).

**Comment by jdkent** [comment]
Hi @sjw42,

That particular issue _should_ be fixed with release `v0.4.3` which should be available in the next half hour, could you try again with that version and see if you run into the same error?

**Comment by sjw42** [comment]
Will do! Thanks very much for fixing so quickly (and for making such a great tool!) I'll give it a test run tomorrow morning and will let you know how it goes.

**Comment by sjw42** [comment]
Just tested v0.4.3 and it worked fine without the -a and -l flags. I also tested a second time with an atlas and lookup table added to see what that output looked like and got a few lines of the error pasted below. I noticed the same error in the sample output on the tutorial pages of the readthedocs site, so maybe these errors can just be ignored? Thanks again for your help!

/opt/miniconda-latest/lib/python3.7/site-packages/nibetaseries/interfaces/nilearn.py:130: RuntimeWarning: invalid value encountered in greater
  n_lines = int(np.sum(connmat > 0) / 2)
